spatialprofilingtoolbox/db/scripts/upload_sync_small.py

```diff
@@ -0,0 +1,85 @@
+"""Synchronize a small data artifact with the database."""
+
+import argparse
+from json import loads as json_loads
+
+from spatialprofilingtoolbox.db.database_connection import get_and_validate_database_config
+from spatialprofilingtoolbox.db.database_connection import DBCursor
+from spatialprofilingtoolbox.workflow.common.cli_arguments import add_argument
+
+from spatialprofilingtoolbox.standalone_utilities.log_formats import colorized_logger
+logger = colorized_logger('upload_sync_small')
+
+APPROVED_NAMES = ('findings', 'gnn_plot_configurations')
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        prog='spt db upload-sync-small',
+        description='Synchronize small lists of strings for each study with the database.'
+    )
+    parser.add_argument(
+        'name',
+        help='The name of the table of strings to be synchronized.',
+    )
```

Comment:
I propose that we make this into a mutually-exclusive argument group (see [docs](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group)), or else provide the [`choices`](https://docs.python.org/3/library/argparse.html#choices) argument, so that the names are available for inspection at the CLI.